### ISSUE TITLE
Check how to avoid overfitting with DDPM in the conditional and unconditional mode

### ISSUE THREAD

**Issue by JdbermeoUZH** [JdbermeoUZH/MastersThesisUIASegmentation#75]
DoD:

Carry the following experiments for the brain and wmh datasets to find appropriate cond+uncond checkpoints:

- Check no meaningful shift between train and validation for small DDPMs with no uncond training (latest and earliest checkpoint):
> - [x] 256x256 imgs, Small Unet (7.5M), large batch size, up to 22.5k steps (overfit to train, too big a loss in val in SD)
> - [x]  256x256 imgs, Large Unet (140M), small batch size, up to 100k steps
> - [x] 128x128 imgs, Small Unet (7.5M), large batch size, up to 50k steps

- Check no meaningful shift between train and validation for small DDPMs with uncond training (latest and earliest checkpoint):
> - [x] 256x256 imgs, Small Unet (7.5M), large batch size
> - [x] 256x256 imgs, Large Unet (140M), small batch size, up to 50k steps
> - [x] 256x256 imgs, Large Unet (140M), large batch size, up to 22.5k steps
> - [ ] 128x128 imgs, Small Unet (7.5M), large batch size
